entry_types/scrolled/app/controllers/pageflow_scrolled/editor/content_elements_controller.rb

```diff
@@ -0,0 +1,51 @@
+module PageflowScrolled
+  module Editor
+    # @api private
+    class ContentElementsController < ActionController::Base
+      include Pageflow::EditorController
+
+      def create
+        section = Section.find(params[:section_id])
```

Comment:
For content elements and sections there is no test ensuring that it's not possible to create models in a different entry. At least here it is thus still possible.